### ISSUE TITLE
What should I do if I just want to test car without retraining

### ISSUE THREAD

**Issue by Deep-learning999** [pythonlessons/TensorFlow-2.x-YOLOv3#21]
What should I do if I just want to test car without retraining，I tried to train car only, but the loss was 10 to 20 and couldn't go down

**Comment by Deep-learning999** [comment]
418/452, lr:0.000007, giou_loss:   2.78, conf_loss:  17.23, prob_loss:   0.13, total_loss:  20.15
epoch: 7 step:  419/452, lr:0.000007, giou_loss:   5.54, conf_loss:  18.36, prob_loss:   0.24, total_loss:  24.14
epoch: 7 step:  420/452, lr:0.000007, giou_loss:   2.87, conf_loss:  17.82, prob_loss:   0.10, total_loss:  20.79
epoch: 7 step:  421/452, lr:0.000007, giou_loss:   2.17, conf_loss:  16.20, prob_loss:   0.12, total_loss:  18.48
epoch: 7 step:  422/452, lr:0.000007, giou_loss:   4.43, conf_loss:  16.48, prob_loss:   0.16, total_loss:  21.07

**Comment by pythonlessons** [comment]
You need to train at least for 100 epochs, 7 epochs doesn't give any good results.
If you would train it for longer, it would work better than extracting cars with pretrained COCO dataset because there is 80 classes...
But if you still want to test that, check my object tracking tutorial https://pylessons.com/YOLOv3-TF2-DeepSort/
Analyze code, and you should check how I used to implement how to detect only "person". You can use it in similar way
